CPT331.iOS/CPT331.iOS/EventCategory.swift

```diff
@@ -0,0 +1,188 @@
+//
+//  EventCategory.swift
+//  CPT331.iOS
+//
+//  Created by Peter Weller on 8/10/2016.
+//  Copyright © 2016 Peter Weller. All rights reserved.
+//
+
+import UIKit
+
+enum EventCategory {
```

Comment:
The event categories are defined by Eventfinda, which i doubt will be changed very often. The main reason i'm using an enum is so that i can easily map the categories to images and their "parent category", so that i can colour code the markers/icons.

Adding a database layer to store data which i could otherwise store in a single file just makes accessing the data more inconvenient for me.